### ISSUE TITLE
Class attribute docstrings show wrong values.

### ISSUE THREAD

**Issue by Nereg** [Yepoleb/python-a2s#50]
So if you get a docstring for an attribute in a class on this library it shows docstring of next attribute:
![image](https://user-images.githubusercontent.com/47698366/226203227-64b834c9-c7f2-4406-b887-2bd2d5b5e6aa.png)
https://github.com/Yepoleb/python-a2s/blob/c09ff09f06bcedc59c0b058865a20a6d66cdb214/a2s/info.py#L16-L33
As you can see it displays docstring value from next attribute (`server_name`) not correct one.
And it violates a part of [PEP 257](https://peps.python.org/pep-0257/#what-is-a-docstring):
```
String literals occurring immediately after a simple assignment at the top level of a module, class, or __init__ method are called “attribute docstrings”.
```
So all of the docstrings in classes should be moved bellow the attributes to display correctly.

**Comment by Yepoleb** [comment]
Thanks for bringing this to my attention.

**Comment by Nereg** [comment]
I could make a PR for that if you need to do something else

**Comment by Yepoleb** [comment]
Yes please. It was already fixed as part of commit 43150c13363cecd32a63d9b5c863a980741ba877 but as you know that got reverted and moved to a branch.
